src/core/src/node.cpp

```diff
@@ -790,26 +790,28 @@ bool ov::Node::constant_fold(OutputVector& output_values, const OutputVector& in
         return false;
 
     NodeVector nodes;
-    TensorVector input_tensors;
+    HostTensorVector input_tensors;
```

Comment:
I disagree with this change because you remove code which works with `ov::Tensor` and replace it by code which uses `ngraph::HostTensor` (but `HostTensor` will be removed soon).

Reply:
The better option is to migrate evaluates from `HostTensor` to `ov::Tensor`

Reply:
ok, i'll do that

Reply:
done. Please take a look